Developer-Guide.md

```diff
@@ -273,7 +281,9 @@ $ popd
 $ rm -rf "${tmpdir}"
 ```
 
-# Update Docker configuration
+# If you want to run Kata Containers with Docker
```

Comment:
I would probably call this something more like `Installing Kata Containers into Docker`

Reply:
To be consistent, I think it should say "Running Kata Containers with [Docker|Kubernetes] for here and for 322

Reply:
Yes I will take @egernst proposal here because we don't really install Kata Containers into Docker. We install Kata Containers so that upper layers like Docker or Kubernetes can use it.

Reply:
@sboeuf @egernst makes sense and that looks good to me :)